django/test/testcases.py

```diff
@@ -887,6 +887,57 @@ def assertXMLNotEqual(self, xml1, xml2, msg=None):
                 standardMsg = '%s == %s' % (safe_repr(xml1, True), safe_repr(xml2, True))
                 self.fail(self._formatMessage(msg, standardMsg))
 
+    def assertFormValid(self, form, msg=None):
+        """
+        Asserts that a form is valid
+        """
+        if not form.is_valid():
```

Comment:
I think it would be nice if these methods verified the value is identical to either `True` or `False` and not some other truthy/falsey value. This would align with recommendation in the [coding style guildelines](https://docs.djangoproject.com/en/3.0/internals/contributing/writing-code/coding-style/):

> Use assertIs(…, True/False) for testing boolean values, rather than assertTrue() and assertFalse(), so you can check the actual boolean value, not the truthiness of the expression.

As well as stdlib [unittest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue):

> Note that this is equivalent to bool(expr) is True and not to expr is True (use assertIs(expr, True) for the latter).

Reply:
Something like this? 
`if form.is_valid() is not True:`

Reply:
Yes, exactly. However, if the value is truthy but not `True`, I think the current message could be confusing. So perhaps some alternate failure message should be used when the method doesn't return a `bool`.

Same idea should be applied to `assertFormNotValid`.